bay/plugins/attach.py

```diff
@@ -23,16 +23,17 @@ def load(self):
 @click.command()
 @click.argument("container", type=ContainerType())
 @click.option("--host", "-h", type=HostType(), default="default")
+@click.option("--shell", "-s", "shell_path", default="/bin/bash")
 @click.argument("command", nargs=-1, default=None)
 @click.pass_obj
-def attach(app, container, host, command):
+def attach(app, container, host, command, shell_path):
     """
     Attaches to a container
     """
     if command:
-        shell = ['/bin/bash', '-lc', ' '.join(command)]
+        shell = [shell_path, '-lc', ' '.join(command)]
```

Comment:
Root's configured `$SHELL` is `/bin/sh` in our containers. So using that would break things as they are. This change maintains existing behavior unless the user specifies a different shell.